python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1607,8 +1618,30 @@ def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList,
         badMaskPlanes.append("CLIPPED")
         badPixelMask = afwImage.Mask.getPlaneBitMask(badMaskPlanes)
 
-        return AssembleCoaddTask.assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList,
-                                          spanSetMaskList, mask=badPixelMask)
+        retStruct = AssembleCoaddTask.assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList,
+                                               spanSetMaskList, mask=badPixelMask)
+
+        # Propagate PSF-matched EDGE pixels to coadd SENSOR_EDGE and INEXACT_PSF
+        # Psf-Matching moves the real edge inwards
+        self.applyAltEdgeMask(retStruct.coaddExposure.maskedImage.mask, spanSetMaskList)
+        return retStruct
+
+    def applyAltEdgeMask(self, mask, altMaskList):
+        """!
+        \brief Propagate alt EDGE mask to SENSOR_EDGE AND INEXACT_PSF planes
+
+        @param mask: original mask
+        @param altMaskList:  List of Dictionaries containing spanSet lists.
+                             Each element contains the new mask plane name
+                             (e.g. "CLIPPED and/or "NO_DATA") as the key,
+                             and list of SpanSets to apply to the mask.
+        """
+        maskSensorEdgeBit = mask.addMaskPlane("SENSOR_EDGE")
+        maskInexactPsfBit = mask.addMaskPlane("INEXACT_PSF")
+        maskValue = 2**maskSensorEdgeBit | 2**maskInexactPsfBit
+        for visitMask in altMaskList:
+            for spanSet in visitMask['EDGE']:
```

Comment:
How do you know `visitMask` has an `EDGE` entry?

Reply:
WarpCompare writes 3 hardcoded alternative masks: CLIPPED, NO_DATA, EDGE. 
https://github.com/lsst/pipe_tasks/compare/tickets/DM-13410#diff-f9cf63fcf2aed29eec448bae6d243b5fR1729  I  put an check in here, but hope that someone doesn't remove EDGE later.